### PR TITLE
doc: release-notes: Updates for 1.6 (TF-M, Crypto, SPM, Fprotect)

### PR DESCRIPTION
**Pull request by oyvindronningstad**
Bugfixes and new features

Signed-off-by: Øyvind Rønningstad <oyvind.ronningstad@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -184,7 +185,33 @@ Zigbee
...
+  * Added functionality to configure and enable crypto hardware acceleration as part of the TF-M build.
+  * Added configurations to enable PSA Crypto APIs in non-TF-M builds.
+  * (EXPERIMENTAL) psa_its: Added a library to provide ITS APIs for using the Zephyr settings subsystem for non-volatile storage of key material. Warning: The storage format is likely to change without backwards compatibility.
+* nrf_cc3xx_platform/nrf_cc3xx_mbedcrypto
+  * Updated the use mbed TLS version 2.26.0 to align with upstream TF-M.
+  * Added low-level APIs for managing and using hardware unique keys located in the KMU peripheral, or flash + K_DR, when no KMU is available.
+  * Added platform APIs for hmac_drbg
+  * Full changelog: `nrfxlib/crypto/nrf_cc3*/CHANGELOG.rst`
+* hw_unique_key: New library for managing and using hardware unique keys, building on the APIs in nrf_cc3xx_platform. HUKs are secret keys that are kept hidden from the application code, but which can be used by the application for deriving keys for other purposes, such as encrypting data for storing.
+
```
Comment (Vge0rge) [inline doc/nrf/releases/release-notes-latest.rst:210]: Let's add the text for the crypto samples under the crypto section, maybe in a "samples" subsection. 

Added samples showcasing the usage of the Platform Security Architecture (PSA) APIs . The samples perform various cryptographic operations such as encryption/decryption using  symmetric and asymmetric ciphers, key exchange,  hashing and random number generation. Both TF-M enabled targets and secure only targets are supported.

Comment (b-gent) [reply doc/nrf/releases/release-notes-latest.rst:210]: will do, please don't push here

Comment (Vge0rge) [reply doc/nrf/releases/release-notes-latest.rst:228]: Yep, I will not push here. Feel free to change the wording if you have any suggestions.

**Comment by oyvindronningstad** [comment]
@Vge0rge @b-gent I pushed known issues as well as some small updates to the release notes.

**Comment by b-gent** [comment]
> @Vge0rge @b-gent I pushed known issues as well as some small updates to the release notes.

Thanks, I'll check as soon as the build completes.

**Comment by b-gent** [comment]
Rebased and resolved conflicts. Also included small fixes from https://github.com/nrfconnect/sdk-nrf/pull/4960 here

**Review by b-gent — APPROVED** [review]

**Comment by b-gent** [comment]
This is ready to merge from my perspective.
It would be preferable to merge this before other release notes updates to avoid any more merge conflicts.

**Review by osaether — APPROVED** [review]
LGTM

**Review by ioannisg — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -200,22 +202,71 @@ Zigbee
...
+* Added support for hardware-accelerated cryptography in TF-M using the Nordic Security module (nrf_security).
+  When enabled (default), any calls to psa_crypto APIs will utilize the CryptoCell hardware on nRF9160 and nRF5340.
+* Added support for using hardware unique keys (HUKs) for key derivation (``TFM_CRYPTO_ALG_HUK_DERIVATION``).
+  TF-M automatically generates and stores random hardware unique keys (if not present), using the :ref:`lib_hw_unique_key` library.
+* TFM_MINIMAL: Added a size-optimized configuration of TF-M which provides a minimal set of features:
+
+  * This configuration requires 32 kB of flash and provides random number generation, SHA-256, and the platform memory read service, which is analogous to the feature set of :ref:`secure_partition_manager`.
+  * The configuration is showcased in the :ref:`tfm_hello_world` sample.
+
```
Comment (Vge0rge) [inline doc/nrf/releases/release-notes-latest.rst:216]: In the TF-M section we can add the changes for the protected storage service: 

The TF-M protected storage service is now using non-static labels when deriving encryption keys. The derivation labels are generated by combining the client id of the requesting service and the UID of the resource.

Comment (ioannisg) [reply doc/nrf/releases/release-notes-latest.rst:216]: @oyvindronningstad could you implement this change, please?

**Comment by b-gent** [comment]
Done in https://github.com/nrfconnect/sdk-nrf/pull/4966

**Pull request closed without merging**